gcloud/monitoring/timeseries.py

```diff
@@ -127,6 +127,7 @@ class Point(collections.namedtuple('Point', 'end_time start_time value')):
     :type start_time: string or None
     :param start_time: An optional start time in RFC3339 UTC "Zulu" format.
 
+    :type value: int
```

Comment:
A scalar, here, means an int, float, bool, or string. A distribution would be represented as a dictionary.

The precise details will evolve.